.circleci/workflows.yml

```diff
@@ -672,9 +663,6 @@ commands:
             echo Total containers $CIRCLE_NODE_TOTAL
 
             if [[ -v MAIN_RECORD_KEY ]]; then
-              if [[ <<parameters.package>> == 'app' && <<parameters.type>> == 'e2e' ]]; then
-                export DEBUG=cypress:*
-              fi
```

Comment:
Removed these DEBUG flag from windows because I couldn't see if the errors were legitimate or not with the debug on (they were not)